src/app/blogs/page.tsx

```diff
@@ -5,18 +5,20 @@ import PagesWrapper from "@/components/layout/PagesWrapper";
 import BlogCard from "@/modules/blogs/BlogCard";
 
 const BlogPage = () => (
-  <PagesWrapper>
-    <TitleHeader
-      title={"Blogs"}
-      description={
-        "Dive into a diverse collection of articles crafted by our community. Join the conversation, engage with other readers, and contribute to our vibrant blogging community."
-      }
-    />
-    <div className="grid grid-cols-1 gap-4 md:grid-cols-3 lg:grid-cols-4">
-      {blogPosts.map((post, index) => (
-        <BlogCard key={index} {...post} />
-      ))}
-    </div>
-  </PagesWrapper>
+  <div className="flex min-h-screen flex-col">
```

Comment:
same with the height here, getting footer to stick to the bottom

Reply:
same as above https://github.com/xplorer-io/website/pull/41/files#r1685551886

Reply:
Updated